curvefs/src/client/s3/disk_cache_base.cpp

```diff
@@ -169,5 +187,26 @@ int DiskCacheBase::LoadAllCacheFile(std::set<std::string> *cachedObj) {
     return 0;
 }
 
+PathType DiskCacheBase::GetPathType(const std::string& path) {
+    struct stat statbuf;
+    if (stat(path.c_str(), &statbuf) != 0) {
+        // stat path error or path not exist
+        // Maybe it was deleted
+        if (errno == ENOENT) {
+            return PathType::kNotExist;
+        }
+        LOG(ERROR) << "stat path error, path = " << path
```

Comment:
`LOG(WARNING)`? Since it's not a critical error and you ignore it in the caller.

Reply:
> `LOG(WARNING)`? Since it's not a critical error and you ignore it in the caller.

fix